integrations/docker/images/base/chip-build/Dockerfile

```diff
@@ -123,6 +123,14 @@ RUN set -x \
     ruff \
     && : # last line
 
+#TODO: this is only added as a workaround to bloaty build failures, remove it once not needed
```

Comment:
Should we have a github issues number in this TODO, so we don't lose track of it

Reply:
Done, I will create a Ticket at Bloaty